### PR TITLE
Output the sg id as a string

### PR DESCRIPTION
**Pull request by mhd999**

### CODE REVIEW

**Review by itsdalmo — CHANGES_REQUESTED** [review]
I think we need to use the splat-syntax here: i.e. `${element(concat(), 0)}`. If not I believe it will fail for non-vpc usages.

File: outputs.tf [hunk 1]
```diff
@@ -23,5 +23,5 @@ output "role_arn" {
 
 output "security_group_id" {
   description = "The ID of the security group."
-  value       = "${aws_security_group.vpc.*.id[0]}"
+  value       = "${element(aws_security_group.vpc.*.id, 0)}"
```
Comment (itsdalmo) [inline outputs.tf]: Think you have to use the splat-syntax including `concat()`'ing with another empty list - `list("")` - for this to work as expected? See example here:
https://github.com/telia-oss/terraform-aws-acm-certificate/blob/master/outputs.tf#L6

**Comment by itsdalmo** [comment]
It's perhaps worth considering to allow egress all by default when we create the SG in this module? This is how e.g. the [terraform-aws-asg](https://github.com/telia-oss/terraform-aws-asg/blob/master/main.tf#L40-L48) module does it (and therefor any module based off an autoscaling group, such as ECS). What do you think @mhd999 and @mikael-lindstrom?

**Comment by mhd999** [comment]
@itsdalmo Agree, should add this to this PR?

**Comment by mikael-lindstrom** [comment]
I think it will be fine since we are using that approach for everything else. Maybe we should only allow outbound for the CIDR of the VPC? If we are using lambda in a VPC it will only be in the private subnets right?

**Comment by mhd999** [comment]
@mikael-lindstrom Right, but still need to egress to the internet as generic solution work for all cases.

File: outputs.tf (hunk 1 above)
Comment (mhd999) [reply outputs.tf]: True, now it works for both cases.

**Comment by itsdalmo** [comment]
> Right, but still need to egress to the internet as generic solution work for all cases.

Second this! The main reason for opening egress all (in my view) is to reach stuff on the internet, not necessarily anything else in the VPC. Only security concern I can think of is that whatever VM, Lambda or container has egress all can be used to exfiltrate data if it is compromised (if it has access to any data).

**Review by itsdalmo — APPROVED** [review]

**Pull request merged**